src/models/Book.ts

```diff
@@ -0,0 +1,69 @@
+import mongoose, { Document, Schema } from "mongoose";
+
+// Define Book Schema
+const bookSchema = new Schema(
+  {
+    title: {
+      type: String,
+      required: true,
+    },
+    googleID: {
+      type: String,
+      required: false,
+    },
+    authors: {
+      type: [String],
+      required: true,
+    },
+    publisher: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    publishedDate: {
+      type: Date,
+      required: true,
+      get: (val: Date) => val.toISOString().split("T")[0], // Format to YYYY-MM-DD
+    },
+    categories: {
+      type: [String],
+      required: true,
+    },
+    imageLinks: {
+      smallThumbnail: {
+        type: String,
+        required: true,
```

Comment:
Are we going to have problems with these images being required since we decided to tackle image links as a stretch goal?

Reply:
changed for required: false for now